docs/api-reference/endpoints/directions/extra-info/waycategory.md

```diff
@@ -6,7 +6,7 @@ Value list for the directions response values in
 $.routes[*].extras.waycategory.values
 ```
 
-The exponential assignment of the values is used for [bit fields](http://eddmann.com/posts/using-bit-flags-and-enumsets-in-java/). One route section may belong to different categories. Hence, a value of ``97`` would indicate a belonging to ``Paved road``, ``Tunnel`` and ``Highway`` (``64`` + ``32`` + ``1`` ).
+The exponential assignment of the values is used for [bit fields](http://eddmann.com/posts/using-bit-flags-and-enumsets-in-java/). One route section may belong to different categories. Hence, a value of 19 would indicate a belonging to Ford, Tollways and Highway (16 + 2 + 1).
```

Comment:
I would argue a combination of "highway" and "ford" is rather unrealistic. Therefore, you might want to omit the value for "ford" from the example.